Sources/Mimus/Mock.swift

```diff
@@ -16,37 +16,34 @@ public enum VerificationMode {
 
 public enum Arguments: ExpressibleByArrayLiteral {
     case any
-    case actual([MockEquatable?])
+    case actual([Matcher?])
     case none
 
-    public init(arrayLiteral elements: MockEquatable?...) {
+    public init(arrayLiteral elements: Matcher?...) {
         self = .actual(elements)
     }
 
-    public typealias ArrayLiteralElement = MockEquatable?
+    public typealias ArrayLiteralElement = Matcher?
 }
 
 /// Protocol used for verifying equality between objects. Mimus delivers support for base Swift types, check out
-/// https://github.com/AirHelp/Mimus/blob/master/Documentation/Using%20Your%20Own%20Types.md if you want to use your own types
-public protocol MockEquatable {
-
-    /// Function verifying whether two MockEquatable objects are actually equal.
+/// https://github.com/AirHelp/Mimus/blob/master/Documentation/Using%20Your%20Own%20Types.md if you want to use your own
+/// types
+public protocol Matcher {
```

Comment:
Since `Mock` is no longer used, maybe rename `filename` as well?

Reply:
Fair point, renaming.

Reply:
Oh, wait, no, taking it back - so this file contains the most important public facing protocols (`Mock` and `Matcher`) so that we have a single place for users to see what can be done with this framework (`Storage` is also public, but its existence is merely an implementation detail for consumers). 

And I think `Mock` is a more common name for users to see, they'll look here first rather than in `Matcher.swift`.